src/components/ReportActionItem/TripDetailsView.tsx

```diff
@@ -125,7 +125,7 @@ function ReservationView({reservation}: ReservationViewProps) {
             onSecondaryInteraction={() => {}}
             iconHeight={20}
             iconWidth={20}
-            iconStyles={[styles.tripReservationIconContainer, styles.mr3]}
+            iconStyles={[styles.tripReservationIconContainer(false), styles.mr3]}
```

Comment:
Perhaps this style should go in StyleUtils, I don't think we use many functions in the styles file. WDYT?